src/app/organizer/Columns.tsx

```diff
@@ -0,0 +1,419 @@
+              >
+                <div className={styles.modPerk}>
+                  <BungieImage src={p.plugItem.displayProperties.icon} />{' '}
+                  {p.plugItem.displayProperties.name}
+                </div>
+              </PressTip>
+            ))}
+        </div>
+      ),
+      disableSortBy: true
+    },
+    {
+      id: 'mods',
+      Header: 'Mods',
+      accessor: (item) =>
+        item.isDestiny2()
+          ? item.sockets?.categories[1]?.sockets
+              .filter((s) => s.plug?.plugItem.collectibleHash || filterPlugs(s))
+              .flatMap((s) => s.plugOptions) || []
+          : [],
+      Cell: ({ cell: { value: plugItems }, row: { original: item } }) => (
+        <div className={styles.modPerks}>
+          {item.isDestiny2() &&
+            plugItems.map((p: DimPlug) => (
+              <PressTip
+                key={p.plugItem.hash}
+                tooltip={<PlugTooltip item={item} plug={p} defs={defs} />}
+              >
+                <div className={styles.modPerk}>
+                  <BungieImage src={p.plugItem.displayProperties.icon} />{' '}
+                  {p.plugItem.displayProperties.name}
+                </div>
+              </PressTip>
+            ))}
+        </div>
+      ),
+      disableSortBy: true
+    },
+    {
+      id: 'stats',
+      Header: 'Stats',
+      columns: statColumns
+    },
+    {
+      id: 'basestats',
+      Header: 'Base Stats',
+      columns: baseStatColumns,
+      show: false
+    },
+    {
+      id: 'masterworkTier',
+      Header: 'Masterwork Tier',
+      accessor: (item) => (item.isDestiny2() ? item.masterworkInfo?.tier : null),
+      sortType: 'basic',
+      sortDescFirst: true
+    },
+    {
+      id: 'killTracker',
+      Header: 'Kill Tracker',
+      accessor: (item) =>
+        item.isDestiny2() &&
+        item.masterworkInfo &&
+        Boolean(item.masterwork || item.masterworkInfo.progress) &&
+        item.masterworkInfo.typeName &&
+        (item.masterworkInfo.progress || 0),
+      Cell: ({ cell: { value }, row: { original: item } }) =>
+        item.isDestiny2() &&
+        (value || value === 0) && (
+          <div title={item.masterworkInfo!.typeDesc ?? undefined} className={styles.modPerk}>
+            {item.masterworkInfo!.typeIcon && <BungieImage src={item.masterworkInfo!.typeIcon} />}{' '}
+            {value}
+          </div>
+        ),
+      sortType: 'basic',
+      sortDescFirst: true
+    },
+    {
+      id: 'masterworkStat',
+      Header: 'Masterwork Stat',
+      accessor: (item) => (item.isDestiny2() ? item.masterworkInfo?.statName : null)
+    },
+    {
+      id: 'notes',
+      Header: 'Notes',
+      accessor: (item) => getNotes(item, itemInfos)
+    },
+    hasWishList && {
+      id: 'wishListNote',
+      Header: 'Wish List Note',
+      accessor: (item) => wishList?.[item.id]?.notes
+    }
+  ]);
+
+  const columnIdMap = new Map();
```

Comment:
Why transform this into a map? We need the actual list of columns because it includes order.